app/screens/settings/notification_auto_responder/notification_auto_responder.tsx

```diff
@@ -66,46 +64,37 @@ const NotificationAutoResponder = ({currentUser, componentId}: NotificationAutoR
     const theme = useTheme();
     const serverUrl = useServerUrl();
     const intl = useIntl();
-    const notifyProps = useMemo(() => getNotificationProps(currentUser), []); // dependency array should remain empty
+    const notifyProps = useMemo(() => getNotificationProps(currentUser), [/* dependency array should remain empty */]);
 
-    const initialAutoResponderActive = useMemo(() => Boolean(currentUser.status === General.OUT_OF_OFFICE && notifyProps.auto_responder_active === 'true'), []); // dependency array should remain empty
+    const initialAutoResponderActive = useMemo(() => Boolean(currentUser.status === General.OUT_OF_OFFICE && notifyProps.auto_responder_active === 'true'), [/* dependency array should remain empty */]);
     const [autoResponderActive, setAutoResponderActive] = useState<boolean>(initialAutoResponderActive);
 
-    const initialOOOMsg = useMemo(() => notifyProps.auto_responder_message || intl.formatMessage(OOO), []); // dependency array should remain empty
+    const initialOOOMsg = useMemo(() => notifyProps.auto_responder_message || intl.formatMessage(OOO), [/* dependency array should remain empty */]);
     const [autoResponderMessage, setAutoResponderMessage] = useState<string>(initialOOOMsg);
 
     const styles = getStyleSheet(theme);
 
     const close = () => popTopScreen(componentId);
 
-    const saveButton = useMemo(() => getSaveButton(SAVE_OOO_BUTTON_ID, intl, theme.sidebarHeaderTextColor), [theme.sidebarHeaderTextColor]);
-
     const saveAutoResponder = useCallback(() => {
-        updateMe(serverUrl, {
-            notify_props: {
-                ...notifyProps,
-                auto_responder_active: `${autoResponderActive}`,
-                auto_responder_message: autoResponderMessage,
-            },
-        });
-        fetchStatusInBatch(serverUrl, currentUser.id);
+        const canSaveSetting = initialAutoResponderActive !== autoResponderActive || initialOOOMsg !== autoResponderMessage;
+
+        if (canSaveSetting) {
+            updateMe(serverUrl, {
+                notify_props: {
+                    ...notifyProps,
+                    auto_responder_active: `${autoResponderActive}`,
```

Comment:
what is the value set when the field is empty?

Reply:
`"false"`